package.json

```diff
@@ -26,14 +26,14 @@
   "dependencies": {
     "chalk": "^2.4.1",
     "cosmiconfig": "^5.0.0",
-    "create-jest-runner": "^0.5.3",
-    "eslint": "^5.6.0"
```

Comment:
Having eslint listed in the dependencies resulted in potentially downloading multiple copies of eslint. Let's rely on the peerDep people should be installing themselves.

Reply:
deduping handles this - it's supposed to be both.

Reply:
No, deduping will not handle this. It will see that this module requires no greater than eslint 5 and my project requires eslint 6 so it will install both.

Reply:
oh sure, that's the issue with what's on master - that's why the RHS of "eslint" must match the peer dep :-)

Reply:
Sorry, I'm not sure I understand. So are your saying the PR is good?

Reply:
No - I'm saying that since the peerDep string is `"^4.0.0 || ^5.0.0 || ^6.0.0"`, `eslint` needs the same exact value in either deps or devDeps.

Putting it in deps means that people don't have to necessarily have eslint installed to invoke it via this runner, which is the goal here. If there's an incompatible eslint installed up the tree, the peer + dep combo will make `npm ls` fail (which everyone should be always running in CI anyways, since if it fails, your dep graph is invalid and you can't rely on anything working)

Reply:
I see what you mean. It's wired because if someone doesn't have eslint installed they'll get a warning in the terminal, but it's your project, so sure.

Reply:
With peer + dep, they won't in fact get a warning - eslint will just be automatically installed and everything will Just Work.